### PR TITLE
WIP: Revamp old eth branch

### PR DESCRIPTION
**Pull request by justinkook**

### CODE REVIEW

File: packages/bitcore-client/src/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Wallet } from './wallet';
```
Comment (micahriggan) [inline packages/bitcore-client/src/index.ts]: we'll keep what's on master in this folder.

File: Dockerfile [hunk 2]
```diff
@@ -17,7 +17,6 @@ COPY ./packages/bitcore-wallet-client/package.json ./packages/bitcore-wallet-cli
 COPY ./packages/bitcore-wallet-service/package.json ./packages/bitcore-wallet-service/package.json
 COPY ./packages/bitcore-node/package.json ./packages/bitcore-node/package.json
 COPY ./packages/insight/package.json ./packages/insight/package.json
-COPY ./packages/crypto-wallet-core/package.json ./packages/crypto-wallet-core/package.json
```
Comment (micahriggan) [inline Dockerfile]: should keep master on this

File: Dockerfile-test [hunk 3]
```diff
@@ -18,7 +18,6 @@ COPY  ./packages/bitcore-wallet-service/package.json ./packages/bitcore-wallet-s
 COPY  ./packages/bitcore-wallet/package.json ./packages/bitcore-wallet/package.json
 COPY  ./packages/insight-previous/package.json ./packages/insight-previous/package.json
 COPY  ./packages/insight/package.json ./packages/insight/package.json
-COPY  ./packages/crypto-wallet-core/package.json ./packages/crypto-wallet-core/package.json
```
Comment (micahriggan) [inline Dockerfile-test]: Same

**Comment by matiu** [comment]
🚀

**Pull request closed without merging**